semeio/jobs/scripts/misfit_preprocessor.py

```diff
@@ -11,14 +11,15 @@
 from semeio.jobs.correlated_observations_scaling.exceptions import EmptyDatasetException
 
 
-class MisfitPreprocessorJob(ErtScript):  # pylint: disable=too-few-public-methods
+class MisfitPreprocessorJob(SemeioScript):  # pylint: disable=too-few-public-methods
```

Comment:
is this pylint comment necessary?

Reply:
Yeah, pylint will complain about too few public methods in the class if the disable statement is not added 😢 We could of course turn it of globally as well, but 🤷